examples/sample-layers/plot-layer/grid-vertex.glsl

```diff
@@ -25,6 +25,8 @@ attribute vec3 normals;
 attribute vec2 instancePositions;
 attribute vec3 instanceNormals;
 
+uniform mat4 modelMatrix;
+uniform mat4 viewMatrix;
 uniform vec3 modelCenter;
```

Comment:
@shaojingli Possible decision point:

* If these uniforms are generated by the `project` package (which is strongly linked to `Viewport.getUniforms`), they could be auto injected by the `project` shader stub. 

* But since they are not actually used by the functions in that package we could just document them as available and require the shader to declare them.

* The `project` package could also just expose a `project_to_viewspace` next to `project_to_clipspace` which would probably fit better with 64 bit support